### ISSUE TITLE
coq-mathcomp-ssreflect package for Coq 8.8.0

### ISSUE THREAD

**Issue by anton-trunov** [coq/opam-coq-archive#296]
With Coq 8.8.0 the following works perfectly:
```shell
opam install --ignore-constraints-on=coq coq-mathcomp-ssreflect.dev
```

**Comment by anton-trunov** [comment]
Thank you for sharing the schedule! We have just released [coq-fcsl-pcm](https://github.com/imdea-software/fcsl-pcm) library for Coq 8.7. And we will do it for Coq 8.8 as well when mathcomp is ready for Coq 8.8. Looking forward to it!

**Comment by CohenCyril** [comment]
Should be solved thanks to #306, right?

**Comment by anton-trunov** [comment]
@CohenCyril Yes, thanks a lot!
